packages/eslint-plugin/docs/rules/array-type.md

```diff
@@ -1,6 +1,6 @@
 # `array-type`
 
-Requires using either `T[]` or `Array<T>` for arrays.
+Require using either `T[]` or `Array<T>` for arrays.
```

Comment:
Oh, interesting! I was actually thinking of going the other way and preferring third-person singular. _"Requires XYZ"_ and such.

Reasoning: imperative mood reads (to me, at least) as a command: that you should do the thing in the sentence. That's only true for some rules, and then within each rule not true for all projects.

OTOH third-person singular indicates it's describing the rule. That's more directly the goal of these description strings & docs.

For reference, https://eslint.org/docs/rules uses imperative but the individual pages such as https://eslint.org/docs/rules/array-callback-return use third-person singular.

Reply:
Ah, maybe I should line up the format with ESLint's, then? Using third-person singular for individual pages.

As for the "command-sounding" tone, though, the imperative mood reads like a description as well. "Require"/"Disallow" etc. sound obvious enough that the omitted subject is "the rule", instead of "you must do X".

Reply:
Yeah I'm thinking it would make sense for ESLint to use third-person singular _everywhere_ 🤷

Reply:
FWIW, I like saving one character😥